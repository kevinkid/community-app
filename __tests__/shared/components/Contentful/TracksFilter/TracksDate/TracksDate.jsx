import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import { TracksDateInner } from 'components/Contentful/TracksFilter/TracksDate';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    theme: {},
  };
  renderer.render(<TracksDateInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
