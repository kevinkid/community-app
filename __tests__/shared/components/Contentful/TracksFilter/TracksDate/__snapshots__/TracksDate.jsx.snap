// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div
  className="undefined "
>
  <span>
    Date
  </span>
  <IconCalendar
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  />
  <DatePicker
    allowFutureYear={false}
    autoFocus={false}
    date={null}
    displayFormat="MMM D, YYYY"
    id="date-picker-start-date"
    numberOfMonths={1}
    onDateChange={[Function]}
  />
  <span>
    -
  </span>
  <IconCalendar
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  />
  <DatePicker
    allowFutureYear={false}
    autoFocus={false}
    date={null}
    displayFormat="MMM D, YYYY"
    id="date-picker-end-date"
    numberOfMonths={1}
    onDateChange={[Function]}
  />
</div>
`;
